ui/src/assets/i18n/de.json

```diff
@@ -0,0 +1,1082 @@
+      }
+    },
+    "create": {
+      "title": "Erstelle Zeitpläne",
+      "requiredName": "Der Name wird benötigt.",
+      "uniqueName": "Der Name existiert bereits.",
+      "duplicateScheduleName": "Doppelter Zeitplan-Name auf dem Formular.",
+      "requiredScheduleName": "Der Name wird benötigt.",
+      "uniqueScheduleName": "Der Name existiert bereits.",
+      "requiredPlatform": "Die Plattform wird benötigt.",
+      "requiredCronExpression": "Der Zeitangaben-Ausdruck wird benötigt.",
+      "arguments": "Argumente",
+      "argumentsDesc": "Argumente können vorangestellt werden mit <code>--</code> (optional). Beispiel:",
+      "invalidArguments": "Eine oder mehr Argumente sind ungültig.<br />Beispiel: <code>myarg=myvalue</code>.",
+      "properties": "Eigenschaften",
+      "propertiesDesc": "Parameter sollten wie folgt starten <code>app.</code> oder <code>scheduler.</code>. Beispiel:",
+      "invalidProperties": "Einer oder mehr Parameter sind nicht gültig.<br />Beispiel: <code>app.myarg=myvalue</code>.",
+      "create": "Erstelle Zeitpläne",
+      "creating": "Erstelle gerade Zeitpläne",
+      "message": {
+        "groupSelectionNotFoundTitle": "Gruppen-Auswahl nicht gefunden.",
+        "scheduleSuccessTitle": "Zeitplan-Erstellung",
+        "schedulesSuccessTitle": "Zeitplan-Erstellung",
+        "scheduleSuccessContent": "Erfolgreiche Zeitplan-Erstellung \"{{ name }}\"",
+        "schedulesSuccessContent": "Es wurden erfolgreich {{ count }} Zeitpläne erstellt"
+      }
+    },
+    "schedule": {
+      "title": "Zeitplan",
+      "taskDetails": "Aufgaben-Details",
+      "deleteSchedule": "Lösche Zeitplan",
+      "taskInformation": "Aufgaben-Informationen"
+    }
+  },
+  "security": {
+    "authenticationRequired": {
+      "title": "Authentifizierung benötigt",
+      "description": "Bitte melden Sie sich an."
+    },
+    "featureDisabled": {
+      "title": "Feature deaktiviert",
+      "description": "Das angefragte Feature ist auf dem Server deaktiviert."
+    },
+    "rolesMissing": {
+      "title": "Rollen fehlen",
+      "description": "Es scheint als ob nötige Rollen fehlen. Bitte kontaktieren Sie ihren Administrator um die Situation zu klären."
+    }
+  },
+  "flo": {
+    "commons": {
+      "invalidValue": "Ungültiger Wert.",
+      "searchForApplications": "Suche nach Anwendungen",
+      "fitToContent": "Dem Inhalt anpassen",
+      "instanceDown": "Anwendungs-Instanz ist nicht ausgerollt oder runter gefahren"
+    },
+    "properties": {
+      "title": "Eigenschaften für",
+      "searchProperties": "Suche Eigenschaften"
+    },
+    "stream": {
+      "invalidAppLabel": "Ungültiges Anwendungs-Etikette.",
+      "invalidStreamName": "Ungültiger Stream-Name.",
+      "uniqueStreamName": "Stream-Name existiert bereits.",
+      "noneOfStreamName": "Stream-Name gibt es bereits auf der Leinwand.",
```

Comment:
Leinwand sounds strange - maybe UI would fit better

Reply:
The EN wording is "Stream name already exists on the canvas." I would prefer "Leinwand" in this case.